symplyphysics/laws/thermodynamics/infinitesimal_work_in_quasistatic_process.py

```diff
@@ -0,0 +1,49 @@
+from sympy import Eq
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## For a process in a closed system, occuring slowly enough for it to be quasi-static (i.e. the system
+## remains in internal physical, but not necessarily chemical, thermodynamic equilibrium), the infinitesimal
+## increment of work done _by_ the system is related to the pressure inside the system and the infinitesimal
+## increment of volume of the system.
+
+# Law: delta W = P * dV
+## delta W - an infinitesimal increment of work done _by_ the system
+## P - pressure inside the system
+## dV - exact differential of an infinitesimal increment of volume of the system
```

Comment:
Here is the same problem.
Maybe we can avoid mentioning differential and simply state that small amount of volume change results in small amount of work done?

Reply:
Oh yeah, I see now. Technically this is to be read as "infinitesimal increment which is an exact differential", but it is a double entendre in here, so I agree this has to go somewhere else. I don't think omitting the information about the type of differentials is a good idea since it is important to tell them apart in thermodynamics.